backend/tracim_backend/tests/functional/test_live_messages.py

```diff
@@ -38,6 +39,52 @@ def big_html_document(workspace_api_factory, content_api_factory, session) -> Co
     return html_document
 
 
+@pytest.fixture
+def big_html_document(workspace_api_factory, content_api_factory, session) -> Content:
+    # NOTE: MySQL and MariaDB have a maximum of 65536 bytes for description,
+    # so the size is chosen accordingly
+    if session.connection().dialect.name in ("mysql", "mariadb"):
+        description = 65000 * "a"
+    else:
+        description = 2000000 * "a"
+
+    return html_document(
+        workspace_api_factory, content_api_factory, session, "Big document", description
+    )
+
+
+def small_html_document(workspace_api_factory, content_api_factory, session, name) -> Content:
+    return html_document(
+        workspace_api_factory, content_api_factory, session, "Small document " + name, name
+    )
+
+
+@pytest.fixture
+def small_html_document_a(workspace_api_factory, content_api_factory, session) -> Content:
+    return small_html_document(workspace_api_factory, content_api_factory, session, "A")
+
+
+@pytest.fixture
+def small_html_document_b(workspace_api_factory, content_api_factory, session) -> Content:
+    return small_html_document(workspace_api_factory, content_api_factory, session, "B")
+
+
+@pytest.fixture
+def small_html_document_c(workspace_api_factory, content_api_factory, session) -> Content:
+    return small_html_document(workspace_api_factory, content_api_factory, session, "C")
+
+
+def put_document(html_document):
+    update_user_request = requests.put(
+        "http://localhost:7999/api/workspaces/{}/html-documents/{}/status".format(
+            html_document.workspace_id, html_document.content_id
+        ),
+        auth=("admin@admin.admin", "admin@admin.admin"),
+        json={"status": "closed-validated"},
+    )
+    assert update_user_request.status_code == 204
```

Comment:
You should put your asserts in the test method because asserts in other code won't be beautified by pytest. Here you could return the `status_code` then assert in the test method.
By the way if this code is not what you are testing, you shouldn't assert and it should be in a fixture.

Reply:
Ok, I have removed the assertion from the fixture, added it back to the test regarding the big document, and the status is not checked anymore for the other tests.